setup.py

```diff
@@ -21,14 +21,15 @@
         'pandas',
```

Comment:
Version lock all the things!

Reply:
I don't quite agree. I envision fnet to be a flexible research tool to be used alongside another tools. Contrast this to if fnet were a standalone tool meant to reside inside a Docker image. In the latter case, I feel it is more appropriate to pin all packages.